tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 minversion = 1.6
 skipdist = True
-envlist = py36,pep8
```

Comment:
CI runs are performed with py36 environment only. Don't change this

Reply:
Done